osbs/build/build_request.py

```diff
@@ -377,17 +377,34 @@ def adjust_for_registry_api_versions(self):
     def adjust_for_triggers(self):
         """
         Remove trigger-related plugins if no triggers set
+        import_image plugin is only removed for non-custom base images
```

Comment:
If 'base' will run import_image, and 'imageA' has opted out of triggers, but 'imageB' has opted in to triggers, imageB will never be triggered.

In this situation, wouldn't it make sense for 'imageA' to run import_image *even though* it does not want to be triggered by its parent?

Or does this not matter because import_image only does useful things for base images (which might have additional tags added outside of the build process)? If that's the case, we should probably explicitly only run import_image for base images...